rules/docker_config.bzl

```diff
@@ -118,7 +118,7 @@ load(
     "@io_bazel_rules_docker//container:container.bzl",
```

Comment:
Move this file to rules/config/docker_config.bzl?

Reply:
And we need to help update abseil's project for this.

Reply:
I dont want to move docker_config precisely for that reason, lets talk about that one more to see if we do it now (though not in this PR), later, or leave it as it is now.

Reply:
I'm ok with leaving it under rules/ as it is. We probably will create a new repository rule version and that can go under rules/config/.